Userland/Libraries/LibGUI/TableView.cpp

```diff
@@ -178,9 +179,16 @@ void TableView::keydown_event(KeyEvent& event)
     if (is_editable() && edit_triggers() & EditTrigger::AnyKeyPressed && (event.code_point() != 0 || is_clear)) {
         begin_editing(cursor_index());
         if (m_editing_delegate) {
-            if (is_delete)
-                m_editing_delegate->set_value(String {});
-            else if (is_backspace)
+            if (is_delete) {
+                if (selection().size() > 1) {
+                    selection().for_each_index([&](GUI::ModelIndex& index) {
+                        begin_editing(index);
+                        m_editing_delegate->set_value(String {});
+                    });
+                } else {
+                    m_editing_delegate->set_value(String {});
+                }
+            } else if (is_backspace)
                 m_editing_delegate->set_value(String::empty());
             else
```

Comment:
Please also add curly braces here. If one branch has them, all of them should have them.